python/hsml/engine/model_engine.py

```diff
@@ -45,15 +45,21 @@ def __init__(self):
 
     def _poll_model_available(self, model_instance, await_registration):
         if await_registration > 0:
+            model_registry_id = model_instance.model_registry_id
             sleep_seconds = 5
             for i in range(int(await_registration / sleep_seconds)):
                 try:
+                    print(model_instance.shared_registry_project_name)
```

Comment:
do you need these print statements? If yes should be logged into a more detailed way that the user can interprete?